src/dialogs.ts

```diff
@@ -56,8 +56,38 @@ export const redWithInvalidFilesErrorDialog = (
         " - `.\\*.reds` (I can fix this to canonical)\n" +
         "\n" +
         "Got:\n" +
-        `${installable.join("\n")}`,
+        `${files.join("\n")}`,
     },
     [{ label: "Ok, Mod Was Not Installed" }],
   );
 };
+export const fallbackInstallerReachedErrorDialog = (
+  api: VortexAPI,
+  log: VortexLogFunc,
+  message: string,
+  files: string[],
+  installable: string[],
+) => {
+  log("error", `Fallback installer: ${message}`, files);
+
+  // It'd be nicer to move at least the long text out, maybe constant
+  // for text + function for handling the boilerplate?
+  api.showDialog(
+    "info",
+    message,
+    {
+      md:
```

Comment:
Nice, it may eventually be redundant but it’s really good to have specific instructions here.

As a minor minor tweak I’d go for more conversational tone but this definitely gets the point across and we might as well do one full pass to match all the messaging :)